drupal/sites/all/features/boinc_standard/boinc_standard.strongarm.inc

```diff
@@ -2049,21 +2049,9 @@ After logging in, you will be redirected to !edit_uri so you can change your pas
   $strongarm->disabled = FALSE; /* Edit this to true to make a default strongarm disabled initially */
   $strongarm->api_version = 1;
   $strongarm->name = 'user_mail_register_no_approval_required_body';
-  $strongarm->value = '!username,
-
-Thank you for registering at !site. You may now log in to !login_uri using the following username and password:
-
-username: !username
-password: !password
-
-You may also log in by clicking on this link or copying and pasting it in your browser:
-
-!login_url
-
-This is a one-time login, so it can be used only once.
-
-After logging in, you will be redirected to !edit_uri so you can change your password.
+  $strongarm->value = '!mailto,
```

Comment:
This is using the email address as the salutation of the message? Seems strange, though I understand that username is not ideal, since that is not something the user normally sees. Can display name not be used here?

Reply:
Unfortunately the BOINC display name is not known the core user's module. Thus the choice to use the user's email address in these emails. It's not a great solution. 

A more sophisticated piece of code would be a boincuser (sub) module that handled the emails to users for registration, changing passwords, etc.